iceoryx_posh/test/moduletests/test_mepoo_memory_manager.cpp

```diff
@@ -274,16 +255,16 @@ TEST_F(MemoryManager_test, getChunkMultiMemPoolTooMuchChunks)
     std::vector<iox::mepoo::SharedChunk> chunkStore;
     for (size_t i = 0; i < ChunkCount; i++)
     {
-        chunkStore.push_back(sut->getChunk(32));
-        chunkStore.push_back(sut->getChunk(64));
-        chunkStore.push_back(sut->getChunk(128));
-        chunkStore.push_back(sut->getChunk(256));
+        chunkStore.push_back(sut->getChunk(32U, PAYLOAD_ALIGNMENT, CUSTOM_HEADER_SIZE, CUSTOM_HEADER_ALIGNMENT));
```

Comment:
What happens when you reverse the order here? I know it is not part of the pull request - it is just for fun :D

Reply:
good question. It's not explicitly tested

Reply:
I added a test to check that the correct `MemPool` is used, even if empty. This should ensure that the order doesn't matter.